pkg/git/git.go

```diff
@@ -1,13 +1,17 @@
 package git
 
 import (
+	"errors"
 	"os"
 	"path/filepath"
 	"strings"
 
+	"github.com/buildsafedev/bsf/pkg/langdetect"
 	"github.com/go-git/go-git/v5"
 )
 
+var ErrFilesNotAddedToVersionControl = errors.New("files are not added to version control")
```

Comment:
Error message should be more descriptive. We should tell the user which file they should add